src/statements/CassandraStatements.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+let moment = require('moment');
+const DATE_FORMAT = 'MM/DD/YYYY HH:mm';
+
+module.exports = {
+
+  /** Create an INSERT prepared statement object.
+   * The prepared statement object has two properties, mutation and params.
+   * mutation: contains the topic INSERT statement
+   * params: contains the params to be binded to the INSERT statement
+   * 
+   * Preparing queries gets the best performance and will ensure that
+   * the JavaScript parameters are correctly mapped to Cassandra types.
+   * 
+   */
+  prepareInsertTopic: topic => {
```

Comment:
Is it really worth to have a file with statements for every situation instead of just having the code that calls Cassandra provide their own statements? I don't see these queries/mutations being shared much across different resolvers, so having all of them in a single file just reduces locality of implementations, adds a file that will grow very large and attract merge conflicts.

Long story short: I'd leave the responsibility of making the correct statements up to the code that call the CassandraClient.